lib/bootloader_setup.pm

```diff
@@ -212,6 +212,11 @@ sub specific_bootmenu_params {
         }
     }
 
+    if (check_var('ARCH', 'aarch64') && check_var('DISTRI', 'sle')) {
+        $args .= " plymouth=0 linuxrc.debug=trace";
+        record_soft_failure "bsc#1005313 - Unhandled level 1 translation fault";
+    }
```

Comment:
It's an interesting idea. We have quite some bugs regarding bootup problems, e.g. see https://bugzilla.suse.com/show_bug.cgi?id=995310
To keep this in in general we could replace the `if (…)` with `if (get_var('DEBUG_BOOTUP'))` and without the soft failure.

Or does anyone have a better idea to only call this if an error has been observed? E.g. reload from last snapshot but with these boot command line options?